packages/tokens/dist/devdot/css/helpers/color.css

```diff
@@ -1,6 +1,6 @@
 /**
  * Do not edit directly
- * Generated on Sat, 16 Jul 2022 14:15:33 GMT
+ * Generated on Wed, 19 Oct 2022 16:34:46 GMT
```

Comment:
[nitpick] Not an issue per se, but for general knowledge: when you run a `yarn build` for the tokens, the dates are updated in every generated file. As a matter or reducing the overall noise, I tend to not commit these files (with only the date changed) but it's OK anyway.

Reply:
not really urgent to address, but how hard would it be to not update these dates if the file hasn't actually changed?